### PR TITLE
Make all 'font-size's and 'line-height's rem

### PR DESCRIPTION
**Pull request by JorikSchellekens**
Font size of the whole app would ideally be controlled by a single
value. This value is currently hard coded using the :root CSS selector.
It is the intention to make this value configurable within riot. In the
interim all font-sizes have been converted to rem by the simple process
of regex. Replacing px values with their equivalent rem values assuming
a font size of 15px and then rounded to three decimal places, which was
the base at the time of this transformation.

This commit doesn't address any scaling issues. I thought it better to
land this unwieldy, mechanical, invisible change before the others
otherwise the pr would be impossible to review thoroughly.

### CODE REVIEW

**Comment by nadonomy** [comment]
Hey @JorikSchellekens, btw you can request reviews from @matrix-org/design and they automatically get put on the design teams radar. :)

For visual changes, we'd normally ask for an ad hoc build to be thrown up somewhere to interact with, but given the px/rem tests we did with @bwindels yesterday these specific changes look reasonable to me. 

> Is design going to start giving us mockups using rem values instead of pixel values?

Design tools aren't this flexible sadly.

**Comment by JorikSchellekens** [comment]
Alright, I'll reformat everything to use 'pixel variables'.

Thanks for the tips!

**Comment by JorikSchellekens** [comment]
Just put all the variables into one place

**Comment by JorikSchellekens** [comment]
Working on getting a test riot hosted for this and in general

File: res/css/_font-sizes.scss [hunk 1]
```diff
@@ -1,3 +1,19 @@
+/*
+Copyright 2020 New Vector Ltd
```
Comment (turt2live) [inline res/css/_font-sizes.scss]: We're copyright `The Matrix.org Foundation C.I.C.` at this layer.

File: res/css/rethemendex.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ cd `dirname $0`
     # we used to have exclude /themes from the find at this point.
     # as themes are no longer a spurious subdirectory of css/, we don't
     # need it any more.
-    find . -iname _\*.scss | fgrep -v _components.scss | LC_ALL=C sort |
+    find . -iname _\*.scss | sort | fgrep -v _components.scss | LC_ALL=C sort |
```
Comment (turt2live) [inline res/css/rethemendex.sh]: Looks like there's a change to the `_components.scss` which needs to be committed here:
image

Comment (JorikSchellekens) [reply res/css/rethemendex.sh]: I'd assumed the autogen was done by yarn. Comited, and removed the useless sort.

**Comment by jryans** [comment]
@JorikSchellekens Is this ready to merge or did you want another look?

**Pull request merged**